docs/README.md

```diff
@@ -1,47 +1,141 @@
+<p align="center"><img src="https://github.com/stackrox/kube-linter/blob/main/images/logo/KubeLinter-horizontal.svg" width="360"></p>
```

Comment:
Did you mean to modify the `README.md` file instead?

Reply:
Also, you can use relative links for the images, GitHub will resolve them. So just "images/logo/..." should work.

Reply:
I tested it successfully on my Git repo before I made the modifications but I could have sworn I used rawusercontent rather than github.com. Can we merge and fix easily online if there are issues with the display for this one? Lost on the first question though. @viswajithiii. I think either way should work simply.